jsapp/js/components/submissions/tableBulkCheckbox.tsx

```diff
@@ -16,7 +16,7 @@ interface TableBulkCheckboxProps {
   onClearSelection: () => void
 }
 
-export function TableBulkCheckbox(props: TableBulkCheckboxProps) {
+export default function TableBulkCheckbox(props: TableBulkCheckboxProps) {
```

Comment:
Was this a biome/linting change or does this affect something down the line

Reply:
It is a biome change :)